### DISCUSSION TITLE
the back-to-top is not displayed on mobile

### DISCUSSION THREAD

**Discussion by leedom92** [vuepress/core#773] [Q&A] [answered]
I find the back-to-top is not displayed on mobile, I think it is nesessary.

<img width="193" alt="1" src="https://user-images.githubusercontent.com/30711792/159615006-27116d29-ae85-42cd-b9f3-126d45106be7.png">

of course, if we just make website for ourselves with vuepress, we can edit the css, but it's  not convenient for the vuepress user who need to read the document. so in my opinion, the back-to-top should be displayed on mobile, hope we can see it in next version.

**Comment by Mister-Hope** [comment, marked as answer]
Feel free to add a new css with `!important` in `.vuepress/styles/index.scss`. 

I would prefer this as personal taste, as in safari , you can always double tap navigation bar to scroll to top.

My theme vuepress-theme-hope is displaying the button on mobile, though.

I think you should agree that display or not is both ok, and it's hard to judge that one behavior is wrong.

**Reply by leedom92** [reply]
yeah, what u said makes sense.
